gum/backend-arm/gumstalker-arm-glue.S

```diff
@@ -9,12 +9,21 @@ _gum_stalker_follow_me:
   .type gum_stalker_follow_me, %function
 gum_stalker_follow_me:
 #endif
-  stmdb sp!, {lr}
+  /*
+   * We need to store the lr onto the stack, so that it can be restored after
+   * returning from __gum_stalker_do_follow_me. However, when calling C code,
+   * the stack is expected to be 8 byte aligned. We therefore push r3 simply as
+   * padding. This is otherwise unnecessary since the calling convention
+   * dictates that r3 is the 4th argument and is therefore caller rather than
+   * callee saved. We don't use r0, as otherwise when restoring the stack, we
+   * would clobber the return value from __gum_stalker_do_follow_me.
+   */
```

Comment:
```suggestion
  /*
   * We need to store LR onto the stack, so that it can be restored after
   * returning from do_follow_me(). However, when calling C code, the stack
   * is expected to be 8 byte aligned. We therefore push R3 simply as padding.
   * This is otherwise unnecessary since the calling convention dictates that
   * R3 is the 4th argument and is therefore caller rather than callee saved.
   * We don't use R0, as otherwise when restoring the stack, we would clobber
   * the return value from do_follow_me().
   */
```